autogpt_platform/docker-compose.yml

```diff
@@ -123,47 +116,6 @@ services:
     environment:
       GOTRUE_MAILER_AUTOCONFIRM: true
 
-  rest:
-    <<: *supabase-services
-    extends:
-      file: ./db/docker/docker-compose.yml
-      service: rest
-
-  realtime:
-    <<: *supabase-services
-    extends:
-      file: ./db/docker/docker-compose.yml
-      service: realtime
-
-  storage:
-    <<: *supabase-services
-    extends:
-      file: ./db/docker/docker-compose.yml
-      service: storage
-
-  imgproxy:
-    <<: *supabase-services
-    extends:
-      file: ./db/docker/docker-compose.yml
-      service: imgproxy
-
-  meta:
-    <<: *supabase-services
-    extends:
-      file: ./db/docker/docker-compose.yml
-      service: meta
-
-  functions:
```

Comment:
Have you tested registering an account? 
Pretty sure we are using functions since you added one.

Reply:
This is for supabase function - deno run typescript stuff. We use postgres functions, which run in postgres